.circleci/config.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     <<: *defaults
     steps:
       - checkout
-      - run: yarn cache clean --force
-      - run: yarn
       - run: yarn config set workspaces-experimental true
+      - run: yarn cache clean --force
+      - run: yarn --frozen-lockfile
```

Comment:
We don't need this as `yarn production-build` does this

Reply:
Removed